mne/time_frequency/tfr.py

```diff
@@ -505,7 +505,8 @@ def _preproc_tfr(data, times, freqs, tmin, tmax, fmin, fmax, mode,
     from ..viz.utils import _setup_vmin_vmax
 
     copy = baseline is not None
-    data = rescale(data, times, baseline, mode, copy=copy)
+    if mode is not None:
+        data = rescale(data, times, baseline, mode, copy=copy)
```

Comment:
> Come to think of it, we should probably make it throw an error if baseline is not None and mode is None to avoid people thinking they're baseline correcting when they're not. And those checks should be done in rescale.

Agreed.